### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by teuben**
clarified CASA/FITS, out of context it looks like glue can read casa images

added a section on comparing images. for a newcomer this was a little harder than it should be.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -90,6 +91,14 @@ How do I make a scatterplot of columns from two different catalogs?
...
 :ref:`Merge <merging>` the two datasets into a single object.
 
+How do I make a scatterplot between two images?
+-------------------------------------------------------------------
+:ref:`Merge <merging>` the two datasets into a single object, like you do for catalogs. This will only work if the two images have exactly the same shape (although it does not check if the WCS is the same).  You can then drop the merged object with the **Image Viewer**, and do this a second time for which you will select the other *Attribute* (on lower left) to be visible. For a third one you select the **Scatter Plot**. You can then select regions in any of the three plots, and the corresponding selections in the other plots will be highlighted. Another interesting way to visualize your two related images is drop a 4th object using **Image Viewer**, but this time select the RGB instead of Monochrome, and put each image into one of the colors. Right mouse down will change the *Contrast* selected color after you have selected the *bias/contrast button* on the top row. Obviously with three images  this is a great way to create a colorful RGB image.
```
Comment (ChrisBeaumont) [inline doc/faq.rst]: Including a screenshot of your two image + scatterplot setup would be helpful here. Also I'd prefer to standardize some of the wording:

"Drop the merged object with the Image Viewer..." -> "Drag the merged dataset into the visualization area to view an image of the first attribute. Repeat a second time to view the second image, and finally create a scatterplot comparing the image intensities at each pixel"

**Comment by ChrisBeaumont** [comment]
looks good to me, but I think a screenshot above would be helpful

**Comment by teuben** [comment]
let me work on a screenshot, I have a nice one.  On 11/02/2015 12:16 PM,
Chris Beaumont wrote:

**Comment by teuben** [comment]
Added screenshot, clarified some text and added figure caption.

**Comment by ChrisBeaumont** [comment]
This looks good to me, the build failures seem unrelated. @astrofrog look good to you?

**Comment by astrofrog** [comment]
@teuben - thanks! Indeed the Travis failures are unrelated (and fixed in master)

**Pull request merged**